rustls/src/keylog.rs

```diff
@@ -160,6 +159,13 @@ impl KeyLog for KeyLogFile {
     }
 }
 
+impl Default for KeyLogFile {
```

Comment:
This is worse. Let's just disable this lint.

Reply:
Fixed.